code/__DEFINES/shuttles.dm

```diff
@@ -13,6 +13,7 @@
 #define EMERGENCY_IDLE_OR_RECALLED (SSshuttle.emergency && ((SSshuttle.emergency.mode == SHUTTLE_IDLE) || (SSshuttle.emergency.mode == SHUTTLE_RECALL)))
 #define EMERGENCY_ESCAPED_OR_ENDGAMED (SSshuttle.emergency && ((SSshuttle.emergency.mode == SHUTTLE_ESCAPE) || (SSshuttle.emergency.mode == SHUTTLE_ENDGAME)))
 #define EMERGENCY_AT_LEAST_DOCKED (SSshuttle.emergency && SSshuttle.emergency.mode != SHUTTLE_IDLE && SSshuttle.emergency.mode != SHUTTLE_RECALL && SSshuttle.emergency.mode != SHUTTLE_CALL)
+#define EMERGENCY_ALMOST_HERE (SSshuttle.emergency && (SSshuttle.emergency.mode == SHUTTLE_CALL) && (SSshuttle.emergency.timeLeft(1) <= 10 MINUTES))
```

Comment:
just check if it's coming and canRecall() returns if false

Reply:
Nah, since canRecall is almost always true (It can only  be false if the shuttle is very nearly here or if an admin force called it). It should check SHUTTLE_CALL, if the shuttle is currently being recalled then the shuttle mode would be SHUTTLE_RECALL and not SHUTTLE_CALL.

Reply:
> Nah, since canRecall is almost always true (It can only be false if the shuttle is very nearly here or if an admin force called it). It should check SHUTTLE_CALL, if the shuttle is currently being recalled then the shuttle mode would be SHUTTLE_RECALL and not SHUTTLE_CALL.

This is not true. canRecall() stops returning true depending on the alert level and the time remaining. "very nearly here" is literally exactly the goal of trying to block it. 10 minutes is arbitrary

Reply:
It shouldn't spawn antags if the shuttle is coming at all, even if they can recall it